### PR TITLE
Express routing

### PR DESCRIPTION
**Pull request by seb879**
Create demo version for deployment for this afternoon, with 1 route test

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    connection.query('SELECT name FROM test', (err, results) => {
+      if (err) {
+        res.status(500).send('Erreur lors de la récupération des ');
+      } else {
+        res.json(results);
+      }
+    });
+  });
+
+/*
```
Comment (erabin05) [inline index.js]: Why does there is all this dead code ??

File: index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+app.use(bodyParser.json());
+// Support URL-encoded bodies
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+
+
+  //All to pick up a filter for the keywords in the database
+  //Test for deployment
+  app.get('/Poi/Filter/', (req, res) => {
```
Comment (erabin05) [inline index.js]: I would avoid uppercase on your urls

File: index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+require('dotenv').config()
+const express = require('express');
+const app = express();
+const port = 3000;
```
Comment (erabin05) [inline index.js]: Think about when you will use react in the same time

File: index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        res.status(500).send('Erreur lors de la récupération des ');
+      } else {
+        res.json(results);
+      }
+    });
+  });*/
+
+app.listen(port, (err) => {
+  if (err) {
+    throw new Error('Something bad happened...');
```
Comment (erabin05) [inline index.js]: "Something bad happend" ?? Like you're cat died ? That is not really specific

**Review by erabin05 — CHANGES_REQUESTED** [review]
- Port 3000 isn't maybe the best choice knowing React use the same
- Choose a way to write your urls for your REST API :  CamelCase, snake_case, kebab-case (https://blog.restcase.com/5-basic-rest-api-design-guidelines/)
- Dead code mustn't be here
- be specific when you throw an error

File: index.js (hunk 1 above)
Comment (seb879) [reply index.js]: done

File: index.js (hunk 2 above)
Comment (seb879) [reply index.js]: done

File: index.js (hunk 3 above)
Comment (seb879) [reply index.js]: done

File: index.js (hunk 4 above)
Comment (seb879) [reply index.js]: done

**Comment by seb879** [comment]
The elements are corrected

File: index.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  extended: true
+}));
+
+
+  //All to pick up a filter for the keywords in the database
+  //Test for deployment
+  app.get('/poi/filter/', (req, res) => {
+    connection.query('SELECT * FROM pointofinterest', (err, results) => {
+      if (err) {
+        res.status(500).send('Erreur lors de la récupération des points d\'intèrêts' );
```
Comment (erabin05) [inline index.js:21]: In english please

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**